docs/ldml/tr35-keyboards.md

```diff
@@ -367,6 +372,12 @@ There are two normalization-safe segments here:
 
 Normalization (and marker rearranging) occurs within each segment.  While `\m{marker1}` is 'glued' to the `\u{0320}`, it is glued within the first segment and has no effect on the second segment.
 
+#### Rationale for 'gluing' markers
+
+It is recognized that the processing described here seems to be an innovation among Unicode normalization implementations.
+
+This specification has markers 'glued' (remaining with) the following character so that if a context ends with a marker, that marker would be guaranteed to remain at the end after processing.  Authors can keep a marker together with a character of interest by emitting the marker just before the character of interest, that is, `output="\m{marker}X"` instead of `output="X\m{marker}"`.
```

Comment:
```suggestion
This specification has markers 'glued' (remaining with) the following character so that if a context ends with a marker, that marker would be guaranteed to remain at the end after processing, consistently located with respect to the next keystroke to be input.  Alternatively, authors can keep a marker together with a character of interest by emitting the marker just before the character of interest, that is, `output="\m{marker}X"` instead of `output="X\m{marker}"`.
```

To be a bit more explicit about the usefulness of end-of-context markers.